scripts/exregional_nonvarcldanl.sh

```diff
@@ -101,6 +101,11 @@ case $MACHINE in
   APRUN="srun"
   ;;
 #
+"ORION")
+  ulimit -s unlimited
+  APRUN="srun"
+  ;;
```

Comment:
It would be nice to keep this the same as the others and add `ulimit -a`. Here and in the other scripts.

Reply:
Done